go.mod

```diff
@@ -6,16 +6,20 @@ require (
 	github.com/agiledragon/gomonkey v2.0.2+incompatible
 	github.com/agiledragon/gomonkey/v2 v2.4.0
 	github.com/deckarep/golang-set v1.8.0
+	github.com/go-logr/logr v1.2.0
```

Comment:
Do we need to add these dependencies before adding controller code?

Reply:
Done. Thanks for the reminder.